src/woo_publications/publications/admin.py

```diff
@@ -49,6 +51,11 @@ class PublicationAdmin(AdminAuditLogMixin, admin.ModelAdmin):
     date_hierarchy = "registratiedatum"
     inlines = (DocumentInlineAdmin,)
 
+    def has_change_permission(self, request, obj=None):
+        if obj and obj.publicatiestatus == PublicationStatusOptions.revoked:
+            return False
+        return True
```

Comment:
uh this is a security bug, since you need to call the default behaviour instead of just giving permission:

```suggestion
        return super().has_change_permission(self, request, obj=obj)
```

`InformationCategoryAdmin` is also affected by this

the current form completely disregards user permissions in the admin, meaning that any authenticated user can make changes!

The default implementation in django has this:

```py
    def has_change_permission(self, request, obj=None):
        """
        Return True if the given request has permission to change the given
        Django model instance, the default implementation doesn't examine the
        `obj` parameter.

        Can be overridden by the user in subclasses. In such case it should
        return True if the given request has permission to change the `obj`
        model instance. If `obj` is None, this should return True if the given
        request has permission to change *any* object of the given type.
        """
        opts = self.opts
        codename = get_permission_codename("change", opts)
        return request.user.has_perm("%s.%s" % (opts.app_label, codename))
```